### DISCUSSION TITLE
Set ownership on upload

### DISCUSSION THREAD

**Discussion by mylogon341** [iterate-ch/cyberduck#14589] [Q&A]
Currently we're able to set the permissions on download/ upload. Is there a place to also change the ownership of an uploaded file also? It'd be really useful to be able to set the ownership of an uploaded file to be the same as the directory. 

Is this already possible, or is this worth putting in the request board? It would save me the hassle of uploading something and then ssh into the machine separately to run `chown` to make sure it's corrected.

Thanks

**Comment by dkocher** [comment]
This is in #11965.

**Comment by mylogon341** [comment]
This looks like it's been a requested feature since 2005 @dkocher. Realistically, it's probably never going to be a thing, right? Is there a technical reason it can't be done?

**Reply by dkocher** [reply]
Would need to check again if support is available in OpenSSH (or other server implementations) possibly using a non-standard extension to SFTP.
